.github/workflows/release-container.yml

```diff
@@ -6,6 +6,10 @@
         - '[0-9]+.[0-9]+.[0-9]+'
   concurrency: keycloak-container-release
 
+  env:
+    QUAY_IMAGE: quay.io/aperuffo/keycloak
```

Comment:
@andreaTP please replace all the references to your personal account to Keycloak

Reply:
ouch 🤦

Reply:
at least you know that I tested it 😛